src/compat/eme/custom_media_keys/old_webkit_media_keys.ts

```diff
@@ -184,8 +185,20 @@ export default function getOldWebKitMediaKeysCallbacks() {
   };
   const createCustomMediaKeys =
     (keyType: string) => new OldWebKitCustomMediaKeys(keyType);
+  const setMediaKeys = (elt: HTMLMediaElement,
+                        mediaKeys: MediaKeys|ICustomMediaKeys|null): void => {
+    if (mediaKeys === null) {
+      return;
+    }
+    if (!isCustomMediaKeys(mediaKeys)) {
```

Comment:
Shouldn't we check specifically for a `OldWebKitCustomMediaKeys`?

Going into that function to set an IE11 Custom MediaKeys would be an error here